src/main/java/opendcs/dao/DacqEventDAO.java

```diff
@@ -59,43 +59,42 @@ public void writeEvent(DacqEvent evt) throws DbIoException
 		String txt = evt.getEventText();
 		if (txt.length() >= 256)
 			txt = txt.substring(0,255);
+
+		StringBuilder q = new StringBuilder();
+		q.append("INSERT INTO " + tableName + "(" + columns + ") VALUES(?,?,?,?,?,?,?,?");
```

Comment:
At first glance, tableName looks like a input string (actually it is final static string).  Perhaps this could be renamed to DACQ_EVENT_TABLE_NAME, in the future.